flask_app/models.py

```diff
@@ -186,6 +187,7 @@ class Session(db.Model, TypenameMixin, StatusPredicatesMixin, HasSubjectsMixin,
         Index('ix_session_start_time_status_lower', start_time.desc(), func.lower(status)),
         Index('ix_session_timespan', 'timespan', postgresql_using='gist'),
         Index('ix_session_delete_at', delete_at, postgresql_where=(delete_at != None)),
+        Index('ix_session_updated_at', updated_at.desc(), postgresql_where=(updated_at != None)),
```

Comment:
@maormarcus we need the index to be `asc()` because our use case would be to look for something newer than the last updated time...